crucible/crucible.cabal

```diff
@@ -72,6 +72,9 @@ library
 
   hs-source-dirs: src
 
+  other-modules:
+    Lang.Crucible.Backend.Assumptions
```

Comment:
Is there a reason not to expose this module?

Reply:
Not really, I don't have a strong sense of when to make modules public vs. private, just opted for private because it exposes less of a surface area for people to rely on (e.g., keeping it private lets us reorganize the code further, so long as we keep exporting it from `Backend`).